<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vray_toolbox</class>
 <widget class="QWidget" name="vray_toolbox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>472</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>windowTitle</string>
  </property>
  <layout class="QVBoxLayout" name="form_vLyt">
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="vrayAttr_lbl">
     <property name="text">
      <string>Vray Attributes</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="vray_form">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="applyTo_lbl">
       <property name="text">
        <string>Apply to:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="applyTo_hlyt">
       <item>
        <widget class="QRadioButton" name="selection_rbtn">
         <property name="text">
          <string>Selection</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="hierarchy_rbtn">
         <property name="text">
          <string>Hierarchy</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="addTo_lbl">
       <property name="text">
        <string>Add to:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="addTo_hlyt">
       <item>
        <widget class="QRadioButton" name="transforms_rbtn">
         <property name="text">
          <string>Transforms</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="shapes_rbtn">
         <property name="text">
          <string>Shapes</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="attributes_lbl">
       <property name="text">
        <string>Attributes:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="subd_chk">
       <property name="text">
        <string>Subdivision</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="subdDisp_chk">
       <property name="text">
        <string>Subdivision and Displacement Quality</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="objID_chk">
       <property name="text">
        <string>Object IDs</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="openSubdiv_chk">
       <property name="text">
        <string>Open SubDiv</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="disp_chk">
       <property name="text">
        <string>Displacement Control</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="attrBut_hlyt">
     <property name="topMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="add_btn">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="rem_bt">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
